File: Fixed Heap Dynamic/ok.java
public class Ok {
    public static void main(String[] args) {
        // Fixed-size array allocated on the heap (Java default behavior)
        int[] arr = new int[7]; // Memory is allocated on the heap; size cannot change after creation

        // Populate the array with values
        for (int i = 0; i < arr.length; i++) {
            arr[i] = i * 5;
        }

        // Display the array contents
        System.out.println("\nFixed Heap-Dynamic Array:");
        for (int num : arr) {
            System.out.print(num + " ");
        }
    }
}

File: Fixed Heap Dynamic/ok.py
def fixed_heap_dynamic():
    # Demonstrating a fixed-size heap-allocated array using Python's array module
    import array
    arr = array.array('i', [0] * 5)  # Fixed length, stored on the heap
    
    # Initialize array values
    for i in range(len(arr)):
        arr[i] = i * 5

    # Display the array contents
    print("\nFixed Heap-Dynamic Array (using array module):")
    print(arr.tolist())

fixed_heap_dynamic()

